Box.V2/Managers/BoxSearchManager.cs

```diff
@@ -63,6 +63,7 @@ public BoxSearchManager(IBoxConfig config, IBoxService service, IBoxConverter co
                                                              
         {
 
+            keyword.ThrowIfNullOrWhiteSpace("keyword");
```

Comment:
https://docs.box.com/reference#searching-for-content  query is REQUIRED.
Can I change value string keyword = null, to string keyword,.?
Do you agree?